Pilot/PilotCommands.py

```diff
@@ -48,40 +54,40 @@ def execute( self ):
     self.log.info( 'Uname      = %s' % " ".join( os.uname() ) ,True, True)
     self.log.info( 'Host Name  = %s' % socket.gethostname() ,True, True)
     self.log.info( 'Host FQDN  = %s' % socket.getfqdn() ,True, True)
-    self.log.info( 'WorkingDir = %s' % self.pp.workingDir ,True, True)  # this could be different than rootPath
+    self.log.info( 'WorkingDir = %s' % self.pilotParams.workingDir ,True, True)  # this could be different than rootPath
 
     fileName = '/etc/redhat-release'
     if os.path.exists( fileName ):
-      f = open( fileName, 'r' )
-      self.log.info( 'RedHat Release = %s' % f.read().strip() )
-      f.close()
+      fFile = open( fileName, 'r' )
```

Comment:
`with` can be used to open (and close, thanks to the "free" context manager) the files.